core/common/src/Code.ts

```diff
@@ -234,12 +234,19 @@ export namespace CodeScopeSpec {
 }
 
 /**
- * The properties of a CodeSpec
- * @public */
+ * The JSON properties of a CodeSpec
+ * @public
+ */
 export interface CodeSpecProperties {
   scopeSpec: {
+    /** the type of CodeSpec */
     type: CodeScopeSpec.Type;
+    /** If true, the federationGuid of the scope element identifies the scope, for Codes that are globally unique.
+     * Otherwise, the ElementId of the scopeElement is used, for Codes that are scoped only within a single iModel.
+     */
     fGuidRequired?: boolean;
+    /** The relationship className, when `type` is `CodeScopeSpec.Type.RelatedElement` */
```

Comment:
In format "Schema:Class"?

Reply:
That's an excellent question. The only example I could find in the iModels in my disk just has a className with no schema prefix. @PaulPierog how does that work?

Reply:
I agree it should be "Schema:Class" but the current Plant applications only set className.

Reply:
Can someone point me to the native code that uses this property? There's no reference to "relationship" in CodeSpec.cpp except a TODO comment.

Reply:
> That's an excellent question. The only example I could find in the iModels in my disk just has a className with no schema prefix. @PaulPierog how does that work?

The Plant applications created the CodeSpecs and they are only used with the ProcessFunctional schema.  I agree it should be Schema:Class.

Reply:
added clarification that the form is "schema:class"

Reply:
Should we document that it might be missing the schema name (probably is in fact, given only Plant appears to set this)?

Reply:
If the schema name is missing the fallback should be the schema identified by the class of the element where the CodeSpec is referenced.

Reply:
Is it (or will it be) actually implemented that way somewhere? Where?

Reply:
> Is it (or will it be) actually implemented that way somewhere? Where?

PlantSight Aggregation and likely OpenPlant Engineering Data Manager both iTwin.js based.

Reply:
I don't think we should mention the Plant past "misuse". Only they care, and only they (in their code) will look at their CodeSpecs. Nothing in any of the core api's currently look at the value. Someone should write the code that does.

Reply:
I want the documentation of our file format to accurately represent the data that's out there, for our own benefit and that of our users. We don't need to talk about misuse or specific applications, but can we document it as @PaulPierog describes -"schema:class" or "class" with schema implicit? If not go ahead and resolve this.